hooks/useDialog.tsx

```diff
@@ -32,6 +40,32 @@ const useDialog = () => {
     setTooltipY(y)
   }
 
+  const getTooltipText = () => {
+    switch (tooltipId) {
+      case 'connect':
+        return `El Niño Estrella is a multimedia experience. The smart album is a limited edition digital
+          box set`
+      case 'leaderboard':
+        return 'Leaderboard'
+    }
+  }
```

Comment:
can you please move this to its own `lib` file?
- actual: function defined in `hook`
- expected: function defined in standalone `lib` file